extensions/ql-vscode/src/codeql-cli/cli.ts

```diff
@@ -1211,13 +1211,19 @@ export class CodeQLCliServer implements Disposable {
     outputPath: string,
     endSummaryPath: string,
   ): Promise<string> {
+    const supportsGenerateSummarySymbolMap =
+      await this.cliConstraints.supportsGenerateSummarySymbolMap();
     const subcommandArgs = [
       "--format=text",
       `--end-summary=${endSummaryPath}`,
       "--sourcemap",
-      inputPath,
-      outputPath,
-    ];
+    ]
+      .concat(
+        supportsGenerateSummarySymbolMap
+          ? ["--summary-symbol-map", "--minify-output"]
+          : [],
+      )
+      .concat([inputPath, outputPath]);
```

Comment:
Is this easier to read?

```suggestion
    ...(supportsGenerateSummarySymbolMap
          ? ["--summary-symbol-map", "--minify-output"] 
          : []),
    inputPath, 
    outputPath
  ]
```

Reply:
Thanks! I suspected there was a better way of doing it, but I just wasn't aware of the `...` operator.